cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java

```diff
@@ -120,7 +120,7 @@ public Event intercept(Event event) {
                 + map.getMiddle() + ")");
```

Comment:
I understand this PR is the fix for a bug existing in just released 1.10.0. Thus, it should be mentioned in the CNR.